### DISCUSSION TITLE
Support for markdown style formatting?

### DISCUSSION THREAD

**Discussion by jazibobs** [nathanhoad/godot_dialogue_manager#309] [Ideas]
Hey Nathan,

Firstly, I absolutely love this tool. I've been working with the dialogue manager for around a week so far and everything has been so easy to use. 

I was just wondering if you had considered adding support for some basic text formatting like italics, bold, etc. using the same syntax as markdown? If it's something you'd be interested in I'd be more than happy to try and build it myself and send over a pull request.

Thanks for making such an easy to use dialogue system for Godot!

Cheers,
Jared

**Comment by nathanhoad** [comment]
Dialogue already supports basic formatting via the RichTextLabel BBCodes 👍

**Reply by jazibobs** [reply]
Ahh amazing. My bad, I should have checked the docs more carefully.

Cheers!
